liferay-workspace/overrides/login-web-resource-bundle-override/src/main/java/com/liferay/samples/login/web/resource/bundle/LoginModuleResourceBundleLoader.java

```diff
@@ -0,0 +1,48 @@
+package com.liferay.samples.login.web.resource.bundle;
+
+import com.liferay.portal.kernel.util.AggregateResourceBundleLoader;
+import com.liferay.portal.kernel.util.CacheResourceBundleLoader;
+import com.liferay.portal.kernel.util.ClassResourceBundleLoader;
+import com.liferay.portal.kernel.util.ResourceBundleLoader;
+
+import java.util.Locale;
+import java.util.ResourceBundle;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author blah
+ */
+@Component(
+	immediate = true,
+	property = {
+		"bundle.symbolic.name=com.liferay.login.web",
+		"resource.bundle.base.name=content.Language",
+		"servlet.context.name=login-web"
+	}
+)
+public class LoginModuleResourceBundleLoader implements ResourceBundleLoader {
+
+	@Override
+	public ResourceBundle loadResourceBundle(Locale locale) {
+		return _resourceBundleLoader.loadResourceBundle(locale);
+	}
+
+	@Reference(
+		target = "(&(bundle.symbolic.name=com.liferay.login.web)(!(component.name=com.liferay.samples.login.web.resource.bundle.LoginModuleResourceBundleLoader)))"
```

Comment:
Invoke level 9 OSGi cast
```suggestion
		target = "(&(bundle.symbolic.name=com.liferay.login.web)(!(resource.bundle.aggregate=*))(!(component.name=com.liferay.samples.login.web.resource.bundle.LoginModuleResourceBundleLoader)))"
```